app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -75,6 +80,19 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         initSession()
     }
 
+    @SuppressLint("RestrictedApi")
+    override fun onResume() {
+        super.onResume()
+        if (session.isYoutubeTV) {
+            // Send key events on the UI thread to clear webview grey screen, see #1865
+            GlobalScope.launch(Dispatchers.Main) {
+                delay(50)
```

Comment:
Nit: I know we're only blocking for 50 MS here, but I'm a little concerned about blocking the main thread.  It seems like it would be better to delay on a background thread then jump back (or post) to dispatch the events.

Reply:
Actually `delay` is suspending, not blocking: 
https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/delay.html

Reply:
Huh, so this launches on the main loop, suspends on a background thread, returns to main, then does its work?  Well, no changes needed then.  Thanks!